### PR TITLE
Amazon Cloud Player script

### PR DESCRIPTION
**Pull request by SteVwonder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +20,7 @@ Installation
 Copyright
 ---------
 
-  - Copyright 2014 Jiří Janoušek <janousek.jiri@gmail.com>
+  - Copyright 2015 Stephen Herbein
```
Comment (jiri-janousek) [inline README.md:23]: Please add your e-mail. Copyright holders must be reached by e-mail.

File: integrate.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2014 Your name <your e-mail>
```
Comment (jiri-janousek) [inline integrate.js:2]: Replace with the same copyright like in README.md.

File: integrate.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    } catch (e) {}
+
+    try {
+        var albumImage = playerRoot.getElementsByClassName("albumImage")[0];
+        track.artLocation = albumImage.src;
+        if (track.album === null)
+            track.album = albumImage.title;
+    } catch (e) {}
+
+    try {
```
Comment (fenryxo) [inline integrate.js]: Why is the try-catch block here?

File: integrate.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        player.setTrack(track);
+    } catch (e) {}
```
Comment (jiri-janousek) [inline integrate.js:99]: Why is the try-catch block here?

**Comment by jiri-janousek** [comment]
Hi @SteVwonder. Thanks for your work. I've added in-line comments for some minor issues and here come other two:
1.  [Increase the icon size in Makefile to 128](https://groups.google.com/d/msg/nuvola-player-devel/a-axnC05sf8/a8hEsBolxiIJ) `ICON_SIZE ?= 128`.
2. You should add support for other national variants.
   -  Original implementation in Nuvola Player 2: [settings.js](http://bazaar.launchpad.net/~fenryxo/nuvola-player/trunk/view/head:/data/nuvolaplayer/services/amazon/settings.js), [integration.js](http://bazaar.launchpad.net/~fenryxo/nuvola-player/trunk/view/head:/data/nuvolaplayer/services/amazon/integration.js)
   - Example how to do it in Nuvola Player 3: [Web apps with a variable home page URL - Multiple Website Variants](http://tiliado.github.io/nuvolaplayer/development/apps/variable-home-page-url.html#multiple-website-variants)

**Comment by jiri-janousek** [comment]
Hi @SteVwonder. Are you going to address issues from this code review or should I adopt your integration script?

**Comment by jiri-janousek** [comment]
I cannot accept this pull request because of incomplete copyright information.

**Comment by SteVwonder** [comment]
Sorry that I abandoned this PR.  My semester at school caught up with me.  I will review everything this week and submit a new PR.

**Pull request closed without merging**